app/scripts/services/images.js

```diff
@@ -213,10 +213,23 @@ angular.module("openshiftConsole")
       return resources;
     };
 
+    var getEnvironment = function(imageStreamImage) {
+      return _.map(_.get(imageStreamImage, 'image.dockerImageMetadata.Config.Env'),
+              function(entry) {
+                var pair = entry.split('=');
+                return {
+                  name: _.head(pair),
+                  value:_.last(pair)
```

Comment:
Not a new problem, but this won't correctly handle env var values with a `=`. Maybe

```js
var index = entry.indexOf('=');
return {
  name: entry.substring(0, index),
  value: entry.substring(index + 1)
};
```